src/partition/growfs.c

```diff
@@ -151,38 +223,39 @@ int main(int argc, char *argv[]) {
         struct statfs sfs;
         int r;
 
-        if (argc != 2) {
-                log_error("This program requires one argument (the mountpoint).");
-                return EXIT_FAILURE;
-        }
-
         log_set_target(LOG_TARGET_AUTO);
         log_parse_environment();
         log_open();
 
-        r = path_is_mount_point(argv[1], NULL, 0);
+        r = parse_argv(argc, argv, &target);
```

Comment:
why not just introduce arg_target as another global variable?

Reply:
If you think it's better, I can convert... But I like not having a global argument, especially for something that's so important. It shows up in backtraces and such and is must nicer to read.

Reply:
dunno, it's just a scheme we followed so far: that the inherently per-process objects that are command line arguments are read into per-process variables, prefixed with "arg_"...